public/content/translations/es/developers/docs/smart-contracts/languages/index.md

```diff
@@ -277,19 +277,19 @@ El siguiente es un contrato simple implementado en Fe.
 ```
 type BookMsg = bytes[100]
 
-contrato GuestBook:
```

Comment:
No idea how this could have gotten translated
This string is in Crowdin but it's hidden so translators can't access it, and also had no suggested translations in Spanish